src/courtroom.cpp

```diff
@@ -5505,6 +5600,20 @@ void Courtroom::on_evidence_button_clicked()
   }
 }
 
+void Courtroom::on_evidence_context_menu_requested(const QPoint &pos)
+{
+  QMenu *menu = new QMenu(this);
+  menu->addAction(QString("Open evidence folder"), this,
+                  [=] {
+    QString p_path = ao_app->get_real_path(VPath("evidence/"));
+    if (!dir_exists(p_path)) {
+        return;
+    }
+  }
+  );
+  menu->popup(ui_evidence_button->mapToGlobal(pos));
+}
+
```

Comment:
Doesn't do anything because why would it after none of the others do anything? Same issues as the rest too.